Sudoku/sudoku.js

```diff
@@ -4,14 +4,163 @@
 //
 
 var Sudoku = function(board) {
-  
+  this.board = board;
 };
 
 Sudoku.prototype.isSolved = function() {
 
 };
 
 Sudoku.prototype.solve = function() {
```

Comment:
This method `solve` is doing too much, and not just solving the game. It occurs to me that you might create a function along the lines of `initializeBoard` to create a new gameboard, `takeUserInput` to allow players to play the game.